airbyte-integrations/connectors/source-gnews/source_gnews/gnews.yaml

```diff
@@ -0,0 +1,59 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: ["articles"]
+  base_requester:
+    url_base: "https://gnews.io/api/v4"
+    http_method: "GET"
+  base_retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+  common_parameters:
+    token: "{{ config['api_key'] }}"
+    lang: "{{ config['language'] }}"
+    country: "{{ config['country'] }}"
+    nullable: "{{ ','.join(config['nullable']) }}"
+    from: "{{ config['from'] }}"
+    to: "{{ config['to'] }}"
```

Comment:
@Xabilahu similar case as New York, what about implementing incremental syncs to this API?